riak_test/yz_rt.erl

```diff
@@ -291,6 +293,37 @@ load_data(Cluster, Bucket, YZBenchDir, NumKeys, Operations) ->
 load_data(Cluster, Bucket, YZBenchDir, NumKeys) ->
     load_data(Cluster, Bucket, YZBenchDir, NumKeys, [{load_fruit, 1}]).
 
+
+-spec write_data(cluster(), bucket(), non_neg_integer(), non_neg_integer()) ->
+    [ok | {error, term()}].
+write_data(Cluster, Bucket, Start, Count) ->
+    ValueGenerator = fun(Key) -> {Key, "text/plain"} end,
+    write_data(Cluster, Bucket, Start, Count, ValueGenerator).
+
+-spec write_data(cluster(), bucket(), non_neg_integer(), non_neg_integer(), fun((non_neg_integer()) -> ok | term()))  ->
```

Comment:
I wonder if this should just go in `rt` instead of `rt_yz`? Seems like it's not necessarily specific to Yokozuna, and might be useful in other tests as well.